### PR TITLE
Test/e2e wasm

### PR DESCRIPTION
**Pull request by duvbell**
## Notes 
- Add e2e tests for contracts that use account as key in mapping `Map<addr, Counter>`

### Counter program 
- State: Map<Addr, Counter>
- Functions
	- increment(): increment counter base on sender address 
		- Note: the first time called increment is to init counter to 0
	- get_count(addr): get the address of the counter 
	
https://github.com/terra-money/core/blob/main/scripts/tests/ibc-hooks/counter/src/state.rs
	
## Passes
- Can call contract as normal, the contract address change to newly  address with bench32 `pica` 
- Function for new addresses working as normal	
	
## Problems
1. CHECK: creator of the contract is not migrate correctly
image

2. The old key value with `centauri` is not migrated yet.
image

3. Newly derive contract address got change -> need function / mechanism to get the correct address of contract for dapps dev
image

### CODE REVIEW

File: scripts/upgrade/v_6_4_6/pre-script.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
-echo "********** Running Pre-Scripts **********"
+echo -e "\n ********** Running Pre-Scripts **********"
 
 BINARY=$1 
-DENOM=${2:-upica}
+DENOM=${2:-pica}
```
Comment (dzmitry-lahoda) [inline scripts/upgrade/v_6_4_6/pre-script.sh:4]: why not PPICA?

**Comment by dzmitry-lahoda** [comment]
is https://github.com/ComposableFi/composable-cosmos/pull/438 included in release or was upgraded before?

renaming will break existing contracts running and will prevent me to deploy new as new contracts are bigger size

**Comment by dzmitry-lahoda** [comment]
do not see wasm size doubling in release diff https://github.com/ComposableFi/composable-cosmos/compare/v3.0.1...v3.0.2


i looked 1 and 2 gov proposal - no nothing opened

**Comment by duvbell** [comment]
Hi @dzmitry-lahoda , this PR is to check the affection of migrating bench32 prefix to the wasm contract. For your concern on wasm size doubling, we will check and having the corresponding update / test on that

**Pull request merged**